.github/labeler.yml

```diff
@@ -55,16 +55,12 @@ HIVE:
   - hive3/**/*
   - hive-metastore/**/*
   - hive-runtime/**/*
+  - hive3-orc-bundle/**/*
 DATA:
   - data/**/*
 SPARK:
```

Comment:
We got new bundles for `{aws,gcp,azure}` should we add labels to those as well?

Reply:
It is already there. 
line: 69, 76, 85

Reply:
@Fokko: Any more comments for this PR?

Reply:
No, this looks good to me 👍